Core/LocalAdmin.cs

```diff
@@ -305,10 +309,24 @@ private void RunScpsl(ushort port)
                 {
                     FileName = scpslExecutable,
                     Arguments = $"-batchmode -nographics -nodedicateddelete -port{port} -console{server!.ConsolePort} -id{Process.GetCurrentProcess().Id}",
-                    CreateNoWindow = true
+                    CreateNoWindow = true,
+                    RedirectStandardOutput = true
                 };
 
                 gameProcess = Process.Start(startInfo);
+
+                gameProcess.OutputDataReceived += (sender, args) =>
+                {
+                    Logger.LogLine($"[SCPSL] {args.Data}");
+                };
+                gameProcess.BeginOutputReadLine();
+
+                gameProcess.Exited += (sender, args) =>
```

Comment:
Afaik LocalAdmin already handles process exit, what's the point of this?

Reply:
LocalAdmin handles only exit of its own process not SCP:SL, so if game crashes you don't know that until you type any command.